docs/docs/how-tos/How to install and configure Keycloak.md

```diff
@@ -0,0 +1,151 @@
+---
+id: configuring-keycloak
+title: How to configure Keycloak and add new users
+description: |
+  This is a how-to for configuring Keycloak and add new users.
+---
+
+## Configuring Keycloak
+
+[Keycloak](https://www.keycloak.org/) is the name of the open-source user management software that's automatically deployed within Nebari. It's used to store the database of all
+users in your Nebari instance, and can provide connectivity to other services such as GitHub/Auth0 single sign-on.
```

Comment:
I think here is a good place to mention the definitions of roles and groups in the context of Keycloak and what it provides. Then later, when they are mentioned, readers will have a general idea. Then at the end of the doc, we go into more details. Does that make sense?

Reply:
uhm, not sure would like to keep it only for a summary of Keycloak. On the other hand, I think the `Adding a Nebari user` might be a good place to add a preview of that, something along the lines of:

> when you create your user using the Nebari Keycloak command, it will receive the basic permissions and roles associated with the `Analyst` group. For more information, check out the [Permissions and Roles] section below.

Reply:
What this doc needs is an intro section - like: what are folks learning? 

it goes from zero to configuring Keycloak, like in the other doc we need to add this at the top